lib/bootstrap_builders.rb

```diff
@@ -1,4 +1,7 @@
 require "bootstrap_builders/engine"
+require "simple_form"
```

Comment:
These new lines shouldn't be required unless SimpleForm is present. What if SimpleForm is loaded after BootstrapBuilders?

What about autoloading the classes instead?